java/registry/src/main/java/dev/sunbirdrc/registry/helper/EntityStateHelper.java

```diff
@@ -70,6 +70,11 @@ void applyWorkflowTransitions(JsonNode existing, JsonNode updated, List<Attestat
         addAttestationStateTransitions(existing, entityName, modified, allContexts, attestationPolicies);
         addOwnershipStateTransitions(existing, entityName, updated, allContexts);
         ruleEngineService.doTransition(allContexts);
+        removePasswordField(entityName, updated);
```

Comment:
lets assume the password field is an inside another field like userDetails, then this will not be removed and it will be stored in the registry
can we get the path where password is from OwnershipAttributes and then remove that path from the json?

Reply:
Got it, will resolve this issue